scripts/test_command

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
What's this file for?

Reply:
A helper to test webhooks locally.  It is not used in the codebase but can be useful when developing.

Reply:
OH sorry I got confused!! This one **is** actually used...

It's used in build tests https://github.com/peopledoc/peon/pull/24/files/e59a15629d1f9d3ddc87839a839b3945f0e86ed7#diff-d11a00bbe76ec38e3ecb244ec651a50aR687. Maybe you didn't see it because the diff is collapsed by GitHub.